Sources/UIKitBackend/UIKitBackend+Container.swift

```diff
@@ -1,24 +1,26 @@
 import SwiftCrossUI
 import UIKit
 
-final class ScrollWidget: WrapperWidget<UIScrollView> {
+final class ScrollWidget: ContainerWidget {
+    private var scrollView = UIScrollView()
     private var childWidthConstraint: NSLayoutConstraint?
     private var childHeightConstraint: NSLayoutConstraint?
 
-    private let innerChild: BaseWidget
+    private lazy var contentLayoutGuideConstraints: [NSLayoutConstraint] = [
+        scrollView.contentLayoutGuide.leadingAnchor.constraint(equalTo: child.view.leadingAnchor),
+        scrollView.contentLayoutGuide.trailingAnchor.constraint(equalTo: child.view.trailingAnchor),
+        scrollView.contentLayoutGuide.topAnchor.constraint(equalTo: child.view.topAnchor),
+        scrollView.contentLayoutGuide.bottomAnchor.constraint(equalTo: child.view.bottomAnchor),
+    ]
 
-    init(child innerChild: BaseWidget) {
-        self.innerChild = innerChild
-        super.init(child: UIScrollView())
-
-        child.addSubview(innerChild)
+    override func loadView() {
+        view = scrollView
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+    }
 
-        NSLayoutConstraint.activate([
-            innerChild.topAnchor.constraint(equalTo: child.contentLayoutGuide.topAnchor),
-            innerChild.bottomAnchor.constraint(equalTo: child.contentLayoutGuide.bottomAnchor),
-            innerChild.leftAnchor.constraint(equalTo: child.contentLayoutGuide.leftAnchor),
-            innerChild.rightAnchor.constraint(equalTo: child.contentLayoutGuide.rightAnchor),
-        ])
+    override func viewWillLayoutSubviews() {
+        NSLayoutConstraint.activate(contentLayoutGuideConstraints)
```

Comment:
`activate` probably only has to be called once at initialisation, and I'm not entirely certain on this but `viewWillLayoutSubviews` probably runs more than once. Could maybe move this to `loadView`?

Reply:
From what I remember, I tried putting it in `loadView`, and it crashed because the view hierarchy was insufficiently assembled at that point. I can double-check if it works in `viewDidLoad`, but ultimately activating the same instance of `NSLayoutConstraint` multiple times is harmless.

Reply:
Confirmed. Moving this to `viewDidLoad` causes the app to crash, with

> Thread 1: "Unable to activate constraint with anchors <NSLayoutXAxisAnchor:0x60000178c7c0 \\"_UIScrollViewLayoutGuide:0x600003b38e00'UIScrollView-contentLayoutGuide'.leading\\"> and <NSLayoutXAxisAnchor:0x60000178c800 \\"UIKitBackend.BaseViewWidget:0x10f912fd0.leading\\"> because they have no common ancestor.  Does the constraint or its anchors reference items in different view hierarchies?  That's illegal."

However, `updateViewConstraints` also works, is called less frequently, and seems more appropriate, so I'll move it to there.